src/views/Profileview.tsx

```diff
@@ -19,8 +17,6 @@ function Profileview() {
   const [user, setUser] = useState('');
   const [rtime, setRtime] = useState('');
   const [tickets, setTickets] = useState('');
-  const [profileURL, setProfileURL] = useState('');
-  const [color, setColor] = useState('');
```

Comment:
This is required for customization so can you explain why you decided to delete it throughout Profileview.tsx?